{{ cookiecutter.repo_name }}/.github/ISSUE_TEMPLATE/release_checklist.md

```diff
@@ -6,26 +6,37 @@ labels: "release"
 assignees: ""
 ---
 
-### PyPI/GitHub release checklist:
+### PyPI/GitHub rc-release preparation checklist:
 
 - [ ] All PRs/issues attached to the release are merged.
 - [ ] All the badges on the README are passing.
 - [ ] License information is verified as correct. If you are unsure, please comment below.
 - [ ] Locally rendered documentation contains all appropriate pages, including API references (check no modules are
-  missing), tutorials, and other human written text is up-to-date with any changes in the code.
-- [ ] Installation instructions in the README, documentation and on the website (e.g., diffpy.org) are updated.
+  missing), tutorials, and other human-written text is up-to-date with any changes in the code.
+- [ ] Installation instructions in the README, documentation, and the website (e.g., diffpy.org) are updated.
 - [ ] Successfully run any tutorial examples or do functional testing with the latest Python version.
 - [ ] Grammar and writing quality are checked (no typos).
+- [ ] Install `pip install build twine`, run  `python -m build` and `twine check dist/*` to ensure that the package can be built and is correctly formatted for PyPI release.
```

Comment:
Instead of @sbillinge running this part, we let the maintainer ensure the package can be  built during the CI as well as it is PyPI compatible (check .rst files..)